### DISCUSSION TITLE
maps.protomaps.com style compatibility with Maputnik?

### DISCUSSION THREAD

**Discussion by diehl** [protomaps/PMTiles#418] [Q&A] [answered]
I've downloaded the latest planet PMTiles build for 6/28/24 and am serving it locally with Martin. I'm not having luck applying a custom style based on OMT to the map. So as a sanity check, I downloaded one of the JSON style specs from maps.protomaps.com, loaded it in Maputnik, and replaced the protomaps source definition with a XYZ URL from the local Martin server. And sadly I'm seeing nothing. Should this in theory work?

**Comment by bdon** [comment, marked as answer]
No, it shouldn't work. maps.protomaps.com is only for displaying the protomaps MapLibre style. Maputnik by default probably loads an OMT style.

If you want a prebuilt OMT planet you can get one from OpenStreetMap Japan's download server: https://tile.openstreetmap.jp/static/

**Reply by diehl** [reply]
Thank you @bdon - Appreciate the quick reply! I'll give this a shot...
